lib/generate-svg.js

```diff
@@ -158,7 +159,7 @@ module.exports = (files = [], options = {}) => {
                     symbol.setAttribute('id', `${item.id}${formatPostfix(options.sprite.generate.symbol)}`);
                     symbol.setAttribute('viewBox', viewbox.join(' '));
                     attributes.forEach((attribute) => {
-                        if ( !['preserveaspectratio'].includes(attribute.name.toLowerCase()) ) {
+                        if ( ![...svgElementAttributes['*'], ...svgElementAttributes.svg].includes(attribute.name) ) {
```

Comment:
Should we maybe also test to make sure that the attributes we're allowing here are also allowed on `svgElementAttributes.symbol` because that's where they will end up on.